### PR TITLE
Completed lc209

### PR DESCRIPTION
**Pull request by kwu25**

### CODE REVIEW

File: kelvinWu/slidingwindow/lc209/lc209.js [hunk 1]
```diff
@@ -1,5 +1,26 @@
+// Problem Statement #
+
+// Given an array of positive numbers and a positive number ‘S,’ find the length of the smallest contiguous subarray whose sum is greater than or equal to ‘S’. Return 0 if no such subarray exists.
+
```
Comment (MelissaPastore) [inline kelvinWu/slidingwindow/lc209/lc209.js:4]: Please make sure you're writing out a high level summary of the approach and/or detailed comments explaining the approach.

File: kelvinWu/slidingwindow/lc209/lc209.js [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+    } else {
+      end++;
+      if (end != arr.length) currentSum += arr[end];
+    }
+    console.log(currentSum);
+    if (currentSum >= s) minLength = Math.min(minLength, end - start + 1);
+  }
+  if (minLength == Number.MAX_SAFE_INTEGER) return 0;
+  return minLength;
 }
```
Comment (MelissaPastore) [inline kelvinWu/slidingwindow/lc209/lc209.js:30]: Your current solution doesn't pass the second test case. Make sure you're testing your code before submitting it.

**Review by MelissaPastore — CHANGES_REQUESTED** [review]
Hey - your solution isn't passing the second test case. Please make sure to test your code before submitting it. If you're stuck on a problem, you can reference the solution and try to reverse engineer it or write a comment explaining that you know your code isn't passing a test and tell us where you're stuck. Can you please update lc209 accordingly and re-submit? 

Thanks,
Melissa

File: kelvinWu/slidingwindow/lc209/lc209.js (hunk 2 above)
Comment (kwu25) [reply kelvinWu/slidingwindow/lc209/lc209.js:30]: > Your current solution doesn't pass the second test case. Make sure you're testing your code before submitting it.

Can you explain further what you meant? I ran this code and it printed out 1 and saw that the provided solution also printed out 1. What number did you see?

**Pull request closed without merging**